internal/services/appconfiguration/app_configuration_data_source.go

```diff
@@ -44,6 +44,11 @@ func dataSourceAppConfiguration() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"public_network_access": {
+				Type:     pluginsdk.TypeString,
```

Comment:
this is a boolean so
```suggestion
			"public_network_access_enabled": {
				Type:     pluginsdk.TypeBool,
```

Reply:
@katbyte, @tombuildsstuff told me that was okay to leave this field as string (check the comment on your previous review), so I am kinda confused...

Reply:
👍 - didn't see the comment. LGTM now then